azure-pipelines.yml

```diff
@@ -1,6 +1,25 @@
 variables:
   DOTNET_SKIP_FIRST_TIME_EXPERIENCE: true
 
+resources:
+  containers:
+  - container: ubuntu_1404_arm_cross_build_image
+    image: microsoft/dotnet-buildtools-prereqs:ubuntu-14.04-cross-e435274-20180426002420
```

Comment:
Is there anywhere to automatically pull the complete image names from? I imagine we'll need to update those pretty regularly

Reply:
No, however, this will at least standardize them between the ci and the official build in one place . Which is general goodness.

Reply:
Consider pulling these into a separate file and including it as a template so we have a dedicate file to update (and maybe sync between repos)